adoc/deployment-aws.adoc

```diff
@@ -93,6 +93,174 @@ requires your credentials. These can be obtained by following these steps:
 * Click menu:Create Access Key[] on the "Security Credentials" tab.
 * Note down the newly created _Access_ and _Secret_ keys.
 
+=== Create IAM Role, Role Policy, and Instance Profile through AWS CLI ===
```

Comment:
The "closing" markers are not necessary in asciidoc. Please remove the `===` at the end.

Reply:
changed